components/VillagerDetailedCard.jsx

```diff
@@ -0,0 +1,43 @@
+import StyledDetailedImage from '../components-styled/StyledDetailedImage';
+import StyledDetailedCard from '../components-styled/StyledDetailedCard';
+import StyledSpan from '../components-styled/StyledSpan';
+import StyledButton from '../components-styled/StyledButton';
+
+export default function VillagerDetailedCard({
+  name,
+  personality,
+  species,
+  gender,
+  imageUri,
+  index,
+  hue,
+  birthday,
+  catchPhrase,
+  bubbleColor,
+  textColor,
+  onClick,
+}) {
+  function handleClose(e) {
+    e.stopPropagation();
+    onClick(false);
+  }
+
+  return (
+    <StyledDetailedCard index={index} hue={hue}>
```

Comment:
Wenn ich das richtig sehe, verwendest du index in der StyledDetailCard nicht, obwohl du es übergibst.